src/stats/stats.h

```diff
@@ -37,6 +37,51 @@ enum StatsMetricFlags {
   STATS_METRIC_ROCKSDB_SEEK,      // Number of calls of seek in rocksdb
   STATS_METRIC_ROCKSDB_NEXT,      // Number of calls of next in rocksdb
   STATS_METRIC_ROCKSDB_PREV,      // Number of calls of prev in rocksdb
+  STATS_METRIC_BLOCK_CACHE_MISS,
+  STATS_METRIC_BLOCK_CACHE_HIT,
+  STATS_METRIC_BLOCK_CACHE_ADD,
+  STATS_METRIC_BLOCK_CACHE_BYTES_READ,
+  STATS_METRIC_BLOCK_CACHE_BYTES_WRITE,
+  STATS_METRIC_BLOOM_FILTER_USEFUL,
+  STATS_METRIC_BLOOM_FILTER_FULL_POSITIVE,
+  STATS_METRIC_BLOOM_FILTER_FULL_TRUE_POSITIVE,
+  STATS_METRIC_PERSISTENT_CACHE_HIT,
+  STATS_METRIC_PERSISTENT_CACHE_MISS,
+  STATS_METRIC_MEMTABLE_HIT,
+  STATS_METRIC_MEMTABLE_MISS,
+  STATS_METRIC_GET_HIT_L0,
+  STATS_METRIC_GET_HIT_L1,
+  STATS_METRIC_GET_HIT_L2_AND_UP,
+  STATS_METRIC_COMPACTION_KEY_DROP_NEWER_ENTRY,         // key was written with a newer value.
+                                                        // Also includes keys dropped for range del.
+  STATS_METRIC_COMPACTION_KEY_DROP_OBSOLETE,            // The key is obsolete.
+  STATS_METRIC_COMPACTION_KEY_DROP_RANGE_DEL,           // key was covered by a range tombstone.
+  STATS_METRIC_COMPACTION_KEY_DROP_USER,                // user compaction function has dropped the key.
+  STATS_METRIC_COMPACTION_RANGE_DEL_DROP_OBSOLETE,      // all keys in range were deleted.
+  STATS_METRIC_COMPACTION_OPTIMIZED_DEL_DROP_OBSOLETE,  // Deletions obsoleted before bottom level due to file gap
+                                                        // optimization.
+  STATS_METRIC_COMPACTION_CANCELLED,                    // If a compaction was canceled in sfm to prevent ENOSPC
+  STATS_METRIC_BYTES_WRITTEN,         // The number of uncompressed bytes issued by DB::Put(), DB::Delete(),
+                                      // DB::Merge(), and DB::Write().
+  STATS_METRIC_BYTES_READ,            // The number of uncompressed bytes read from DB::Get().  It could be
+                                      // either from memtables, cache, or table files.
+                                      // For the number of logical bytes read from DB::MultiGet(),
+                                      // please use NUMBER_MULTIGET_BYTES_READ.
+  STATS_METRIC_NUMBER_DB_SEEK_FOUND,  // The number of calls to seek/next/prev that returned data
+  STATS_METRIC_NUMBER_DB_NEXT_FOUND,
+  STATS_METRIC_NUMBER_DB_PREV_FOUND,
```

Comment:
duplicated, please take a look at the old metrics we have.

Reply:
Do you mean NUMBER_DB_SEEK?

Reply:
Yes, I think those three metrics should no mean a lot for us.